x-pack/metricbeat/module/meraki/device_health/_meta/fields.yml

```diff
@@ -0,0 +1,299 @@
+- name: device
+  type: group
+  fields:
+    - name: serial
+      type: keyword
+      dimension: true
+    - name: imei
+      type: keyword
+    - name: lan_ip
+      type: ip
+    - name: location
+      type: geo_point
+    - name: mac
+      type: keyword
+    - name: model
+      type: keyword
+    - name: name
+      type: keyword
+    - name: network_id
+      type: keyword
+    - name: notes
+      type: text
+    - name: product_type
+      type: keyword
+    - name: tags
+      type: keyword
+    - name: performance_score
+      type: double
+      metric_type: gauge
+    - name: license
+      type: group
+      fields:
+        - name: activation_date
+          type: keyword
+        - name: claim_date
+          type: date
+        - name: duration_in_days
+          type: long
+        - name: expiration_date
+          type: date
+        - name: head_license_id
+          type: keyword
+        - name: id
+          type: keyword
+        - name: license_type
+          type: keyword
+        - name: order_number
+          type: keyword
+        - name: seat_count
+          type: long
+        - name: state
+          type: keyword
+        - name: total_duration_in_day
+          type: long
+    - name: high_availability
+      type: group
+      fields:
+       - name: enabled
+         type: boolean
+       - name: role
+         type: keyword
+    - name: status
+      type: group
+      fields:
+        - name: gateway
+          type: ip
+        - name: ip_status
```

Comment:
I don't see any field ip_type getting the values. 
Is this meant to be ip_status?

Reply:
great catch - thanks